assessment.md

```diff
@@ -202,30 +203,33 @@ when creating python programs.
   * A = A few status badges are in the README but are not in an
     appropriate location
   * G = Most relevant status badges are in the README and well located
-  * E = All relevant and many useful status badges are in the README and well located
+  * E = All relevant and many useful status badges are in the README and well
+  located
 
 ### Foundations of Software Engineering
 
 * Requirements engineering
-  * N = No effort to document, define, or maintain requirements present
+  * N = No effort to document, define, or maintain requirements present is
+  evident
   * I = Brief or unspecific documentation, definition, and maintenance given
   * A = Explains what requirements have been specified in enough detail
     to convey the most important information
-  * G = Documentation and definitions are well written and gives other team members
+  * G = Documentation and definitions are well written giving other team members
     an informed idea of the process
-  * E = The software engineer displayed highly detailed, elaborate, and complete
+  * E = Displayed highly detailed, elaborate, and complete
     documentation, definition, and maintenance in the given design process
 * Software architecture
   * N = There is no consideration to the structure or organization of the project
   * I = There are small amounts of structure to the system
-  * A = There is a blueprint for the software system that lacks completion
+  * A = There is a blueprint for the software system that is not complete
```

Comment:
Possibly should be reworded to use `incomplete blueprint`.

Reply:
Fixed